sensu/plugins/check-neutron-l3-routers.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+#
+# Calls neutron agent api
+# Checks for routers that have multiple active or all down l3 agents
+#
+# return CRITICAL if any routers found with more than one active l3 agent
+# return CRITICAL if any routers found with all l3 agents standby
+#
+# Dean Daskalantonakis <ddaskal@us.ibm.com>
+
+import sys
+import os
+
+from keystoneauth1 import identity
+from keystoneauth1 import session
+from neutronclient.v2_0 import client as neutron_client
+
+username = os.environ['OS_USERNAME']
+password = os.environ['OS_PASSWORD']
+project_name = os.environ['OS_TENANT_NAME']
+auth_url = os.environ['OS_AUTH_URL']
+
+auth = identity.Password(username=username,
+                         password=password,
+                         project_name=project_name,
+                         auth_url=auth_url)
+
+sess = session.Session(auth=auth)
+neutron = neutron_client.Client(session=sess)
+
+STATE_OK = 0
+STATE_WARNING = 1
+STATE_CRITICAL = 2
+MAX_ROUTERS = 100
+
+
+def check_router(router, router_agents):
+
+    active_agent = False
+    bad_agents = False
+
+    for agent in router_agents['agents']:
+        if 'ha_state' not in agent.keys():
+            print("WARNING: No HA state for l3 agent %s" % agent['id'])
```

Comment:
Here you are issuing a warning but not bubbling up the warning.  Perhaps a better way to do this would be to return the const itself.